pkg/process/config/config.go

```diff
@@ -30,6 +30,8 @@ var (
 	defaultSystemProbeSocketPath = "/opt/datadog-agent/run/sysprobe.sock"
 	// defaultSystemProbeFilePath is the default logging file for the system probe
 	defaultSystemProbeFilePath = "/var/log/datadog/system-probe.log"
+	// defaultWINProbePath is the default TCP endpoint used for connecting to the system probe on windows
+	defaultWINProbePath = "http://localhost:3333/"
```

Comment:
This should probably just be a part of `defaultSystemProbeSocketPath`, which has different values depending on build tag